File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        String strX= Integer.toString(x);
        int left=0;
        int right=strX.length()-1;
        while(left<right){
            if (strX.charAt(left)!=strX.charAt(right))
            return false;
            else {
                left++;
                right--;
            }
        }
        return true;
  

   
}

}

File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        strX=str(x)
        leftptr=0
        rightptr=len(strX)-1
        while(leftptr<rightptr):
            if strX[leftptr]!=strX[rightptr]:
                return False 
            else:
                leftptr+=1
                rightptr-=1
        return True


        